app/lib/services/playmusic.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const PlayMusic = require('playmusic');
+const Promise = require('bluebird');
+
+module.exports = Promise.promisifyAll(new PlayMusic());
```

Comment:
I took a look at the docs for this package and it was a bit confusing. But from the looks of it, it seemed like I only need to call `init` or `login` once and then you can call the other functions. If that's the case, then take means it's saving the credentials on the instance. 

Which brings me to my question: will your users be logging in to Google Play or will _you_ be logging into Play? If it's the former, then for each each user, you'll have to have a separate instance of `PlayMusic`. If it's the latter, you should initialize everything before the server starts so it's ready to go.

Again, I'm not 100% sure it saves the creds. You'll have to double check.